Sources/Sentry/SentryScreenshot.m

```diff
@@ -38,6 +45,27 @@ @implementation SentryScreenshot
     return result;
 }
 
+- (void)saveScreenShots:(NSString *)path
+{
+    NSArray<UIWindow *> *windows = [SentryDependencyContainer.sharedInstance.application windows];
+
+    for (UIWindow *window in windows) {
```

Comment:
`m`: Can't we call `appScreenshots` and then save the data to disk to avoid duplication?

Reply:
We can't use appScreenshots because it try to use the main thread to take the screenshots, and the main thread is most likely blocked during crash. 
But I can refactor both to share some code.